$NetBSD: patch-am,v 1.1.1.1 2006/07/25 16:28:48 salo Exp $

add needed header file for NetBSD, plus fix compiler warning

--- parser/sdp.c.orig	2004-12-14 18:45:45.000000000 +0000
+++ parser/sdp.c
@@ -30,6 +30,7 @@
  */
 
 #include <sys/types.h>
+#include <sys/time.h>
 #include <netinet/in.h>
 #include <inttypes.h>
 #include <stdio.h>
@@ -527,7 +528,7 @@ static void ssa_rsp(int level, uint16_t 
 
 void sdp_dump(int level, struct frame *frm)
 {
-	sdp_pdu_hdr *hdr = frm->ptr;
+	sdp_pdu_hdr *hdr = (void *)frm->ptr;
  	uint16_t tid = ntohs(hdr->tid);
 	uint16_t len = ntohs(hdr->len);
 
